scripts/env.sh

```diff
@@ -1,10 +1,23 @@
 #!/usr/bin/env bash
+
+# hack for buildkite
+if [[ "${CI:-}" == "1" || "${CI:-}" == "true" ]]; then
+  if [[ $(uname -s) == 'Darwin' ]]; then
+    export BUN_INSTALL="$HOME/.bun"
+    export PATH="$BUN_INSTALL/bin:$PATH"
+    export PATH="$(brew --prefix llvm@16)/bin:$PATH"
```

Comment:
this is going to be really hard to debug why darwin in CI is stuck on llvm 16 in the future